src/rgw/rgw_rados.h

```diff
@@ -395,8 +396,172 @@ template <class K, class V>
 class lru_map;
 using tombstone_cache_t = lru_map<rgw_obj, tombstone_entry>;
 
+struct get_obj_io {
```

Comment:
I'm unsure if we would do anything with these helper types outside of d3n, so maybe namespace these?

Reply:
removed during refactorings